pm_uep_opt_write/pm_uep_opt_write.xml

```diff
@@ -1,13 +1,14 @@
 <tool id="pm_uep_opt_write" name="Generate uep outputs (pm-uep-opt)" version="0.1.0" python_template_version="3.5">
     <requirements>
+        <requirement type="package">pymuon-suite</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
         pip show pymuon-suite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
         unzip $folder &&
         cp $charge_density $charge_density.name &&
         cp $castep_log $castep_log.name &&
         out_folder="`python ${__tool_directory__}/get_out_folder.py`" &&
-        python ${__tool_directory__}/torelative.py \$out_folder &&
+        find \$out_folder -type f -name "*.yaml" | xargs sed -i "s#^chden_path: .*#chden_path: .\/#g" &&
```

Comment:
TIL you can use any delimiter for the sed s. This is nice!

Reply:
it's really handy with escape characters :)